redo/__init__.py

```diff
@@ -56,21 +56,32 @@ def retrier(attempts=5, sleeptime=10, max_sleeptime=300, sleepscale=1.5, jitter=
         ...     print "max tries hit"
         max tries hit
     """
+    if jitter > sleeptime:
+        # To prevent negative sleep times
+        raise Exception('jitter (%i) must be less than sleep time (%i)', jitter, sleeptime)
+
+    sleeptime_real = sleeptime
```

Comment:
Minor nit: sleeptime_real vs. sleeptime isn't the easiest to read. How about using "base_sleeptime" or "initial_sleeptime" for the current "sleeptime", and then "sleeptime" or "actual_sleeptime" for the current "max_sleeptime".